feat(indexer): add evm block height -> da height indexer (#214)
celestiaorg/celestia-zkevm-ibc-demo: indexer/README.md

# EVM to Celestia Inclusion Block Indexer

## Overview

> This service indexes the mapping between EVM block heights and Celestia
> inclusion block heights in a [BeaconKit Rollkit
> rollup](https://github.com/rollkit/beacon-kit/tree/rollkit).

Reviewer: ```suggestion
This service indexes the mapping between EVM block heights and Celestia block heights for a [BeaconKit Rollkit
rollup](https://github.com/rollkit/beacon-kit/tree/rollkit).
```
